api-specs/overlays/docs/docs.md

```diff
@@ -0,0 +1,25 @@
+# How to use overlays to transform an API spec
+
+The function of overlays is to apply a transformation to an API spec. The files in this directory are used to transform the Kong Gateway EE and OSS API specs. We are using overlays to decouple docs changes from the auto-generated API spec. 
+
+
+## Installation
+
+We are using this [project](https://github.com/lornajane/openapi-overlays-js) for generation:
+
+* Clone [this](https://github.com/lornajane/openapi-overlays-js) repo, `cd` into the directory where you cloned it. 
+* Run `npm install` to get the dependencies
+* Run `npm install -g` to get `overlayjs` as a command
+
+
+## Apply a transformation
+
+To apply an overlay to a spec use the following command: 
+
+`overlayjs --openapi openapi.yml --overlay overlay.yaml`
```

Comment:
What's the purpose of showing how to output to the console? Won't we always be piping this output to a file?